src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -487,6 +497,22 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
           </div>
         )
       },
+      {
+        id: 'rank',
+        labelText: 'Rank',
+        isChecked: rank,
+        onChange: toggleRank,
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>
+            Rank graph nodes based on configurable criteria such as 'number of inbound edges'. These rankings can be
+            used in the graph find/hide feature to help highlight the most important workloads, services, and
+            applications. Rankings are normalized to fit between 1..100 and nodes may tie with each other in rank.
+            Ranking starts at 1 for the top ranked nodes so when ranking nodes based on 'number of inbound edges', the
+            node(s) with the most inbound edges would have rank 1. Node(s) with the second most inbound edges would have
+            rank 2.
```

Comment:
This is much more clear.  I would maybe change the end to add a little more information:

From: `Node(s) with the second most inbound edges would have rank 2.`

To: `Each selected criteria contributes equally to a node's ranking.  Although 100 rankings are possible, only the required number of rankings are assigned, starting at 1. `